polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java

```diff
@@ -1130,18 +1130,13 @@ public void doRefresh() {
             MAX_RETRIES,
             metadataLocation -> {
               FileIO fileIO = this.tableFileIO;
-              boolean closeFileIO = false;
-              PolarisResolvedPathWrapper resolvedStorageEntity =
-                  resolvedEntities == null
-                      ? resolvedEntityView.getResolvedPath(tableIdentifier.namespace())
-                      : resolvedEntities;
               String latestLocationDir =
                   latestLocation.substring(0, latestLocation.lastIndexOf('/'));
               fileIO =
                   refreshIOWithCredentials(
                       tableIdentifier,
                       Set.of(latestLocationDir),
-                      resolvedStorageEntity,
+                      resolvedEntities,
```

Comment:
if `resolvedEntities` is `null` it will always go to if [block](https://github.com/polaris-catalog/polaris/blob/692203d1a9b76b1d335f6a4b829a58395d6c67cf/polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java#L1125).